src/pages/Login-SignIn/Login.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import API_URLS from '../../config';
+import './Login-SignIn.scss';
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      hidden: false,
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('click', this.closeModal);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('click', this.closeModal);
+  }
+
+  closeModal = e => {
+    if (e.target === document.body) {
+      this.setState({ hidden: true });
+    }
+  };
+
+  requestLogin = e => {
+    e.preventDefault();
+    fetch(API_URLS.SIGNIN, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(res => {
+        if (res.status === 200) {
+          return res.json();
+        }
+      })
+      .then(res => {
+        if (res) {
+          // save localstroage
+          // localStorage.setItem('TOKEN', res.token);
+          // push to main
+          // this.props.history.push('/');
```

Comment:
backend에서 아직 토큰 안나오셔서 요렇게 주석으로 의사코드 남겨놓으신거죠??
만약 그렇다면 아래와 같이 수정해주시면 다른사람도 더 잘 알아볼 수 있고 나중에 전체검색으로 ToDo 검색하면 해당 라인으로 이동할 수 있어서 편할 것 같습니다!
```suggestion
        if (res) {
          // ToDo: 백엔드 토큰 구현되면 추가할 예정
          // save localstroage
          // localStorage.setItem('TOKEN', res.token);
          // push to main
          // this.props.history.push('/');
```